lib/ht/search_client/version.rb

```diff
@@ -1,5 +1,5 @@
 module Ht
   module SearchClient
-    VERSION = '1.12.0'
+    VERSION = '1.12.1'
```

Comment:
We are not changing the interface or adding features, but I have no problem bumping the minor.